torch/csrc/jit/runtime/register_special_ops.cpp

```diff
@@ -406,7 +406,14 @@ RegisterOperators reg({
           push(stack, at::zero_(tensor));
         },
         aliasAnalysisFromSchema()),
-
+    Operator(
+        "aten::is_grad_enabled() -> bool",
+        [](Stack* stack) { push(stack, torch::GradMode::is_enabled()); },
+        aliasAnalysisFromSchema()),
+    Operator(
+        "aten::set_grad_enabled(bool val) -> ()",
+        [](Stack* stack) { torch::GradMode::set_enabled(pop(stack).toBool()); },
```

Comment:
Pretty sure that returning `-> ()` in the schema will return an empty tuple. I think you have to push that or make it return None and return a None

Reply:
Oops, I thought this was the schema for returning nothing.

Reply:
Looks like `prim::Print` uses `-> ()` to denote no return value and pushes nothing to the stack.

Reply:
Yea, i'm not sure that's correct though. If you `print(torch.set_grad_enabled(True)` you'll get `None`, you could just return None here and add that to the stack.

Reply:
Done.